spec/lib/matroid/detector_spec.rb

```diff
@@ -0,0 +1,128 @@
+describe Matroid::Detector do
+  # Keys generated specifically for the tests. Should be removed in the future
```

Comment:
The person testing this gem should already have client keys and a Matroid account.